From 9435f2a81f3fc695e75abc6df744d4aec86f2c47 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 26 Jun 2026 22:25:13 +0800
Subject: [PATCH 4/4] AOSCOS: kicker: drop badge label for new applications

On the kicker menu, since the items are compact, the "New!" badge would
cause the label to expand beyond the regular item height, making the menu
look inconsistently spaced.

Drop the text from within the badges so it would show up as a green dot,
which is good enough and should look more consistent.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 applets/kicker/ItemListDelegate.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/applets/kicker/ItemListDelegate.qml b/applets/kicker/ItemListDelegate.qml
index ddaeb1437c..9052446279 100644
--- a/applets/kicker/ItemListDelegate.qml
+++ b/applets/kicker/ItemListDelegate.qml
@@ -92,7 +92,7 @@ ItemAbstractDelegate {
             active: item.isNewlyInstalled ?? false
 
             sourceComponent: Kirigami.Badge {
-                text: item.hasChildren ? "" : Accessible.name
+                text: ""
                 type: Kirigami.Badge.Type.Positive
                 Accessible.name: i18nc("Newly-installed app, badge, keep short", "New!") // qmllint disable unqualified
                 Accessible.description: item.hasChildren ? i18nc("@info:whatsthis Accessible description for badge", "There is a newly-installed application in this category") // qmllint disable unqualified
-- 
2.52.0

